openreview/venue_request/venue_request.py

```diff
@@ -1617,6 +1617,12 @@ def setup_request_form(self):
                 'order': 47,
                 'required': False,
                 'hidden': True
+            },
+            'reviewer_assignment_quota': {
+                'value-radio': [3, 4, 5],
```

Comment:
AFAIK the API 1 don't support numbers, the UI always send a string. You can check this posting a request form from the UI.

Also value from 3 to 5 is too restrictive. We can use a value-regex that allows any number but you need to convert the string to a number.